### DISCUSSION TITLE
ccf-validation-tools/Eye/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#219] [General]
# ccf-validation-tools/Eye/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Eye/comments/

**Comment by emquardokus** [comment]
Corrected in the current Eye_v1.3_DRAFT table: https://docs.google.com/spreadsheets/d/1tZ3YZ_PJ5viU2h8Bx3l02uhcMZ41AW4sbmJrKfRtfgg/edit#gid=695483621
**Corrected different labels:**
1. In row 52, pigmented layer of retina UBERON:0001782 from  ASCT+B table is retinal pigmented epithelium.
2. In row 38, lens of camera-type eye UBERON:0000965  from ASCT+B table is lens.
There were many more instances in the actual ASCT+B table, corrected all--rows 40-47 and row 53
3. In row 34, **and 50 and 53** the term vasculature of retina UBERON:0004864 from ASCT+B table is retinal vasculature.
4. In row 50, the term pericyte CL:0000669 from ASCT+B table is pericyte cell.

**Comment by emquardokus** [comment]
Corrected in the current Eye_v1.3_DRAFT table: https://docs.google.com/spreadsheets/d/1tZ3YZ_PJ5viU2h8Bx3l02uhcMZ41AW4sbmJrKfRtfgg/edit#gid=695483621
Blank ontology ID updates:
Ro2 14 Glycinergic Amacrine Cell	glycinergic amacrine cell	CL:4030028
Row 15 medium-field cell to medium field retinal amacrine cell	CL:0004252
Row 15 wide field cell	wide field retinal amacrine cell	CL:0004253
Row 18 A2 cell	A2 amacrine cell	CL:0004219
Row 19 starburst amacrine cell	found starburst amacrine cell	CL:0004232

**Comment by aleixpuigb** [comment]
New CL term:
CL:4033015 retinal astrocyte

**Reply by emquardokus** [reply]
Where does this appear in table? Do not see it.

**Reply by aleixpuigb** [reply]
It is not in the table, but as it could be relevant for the authors, I wanted to inform of its creation in case they would like to add it in the future.

**Comment by aleixpuigb** [comment]
## New CL term:
eye 21 : ON-blue cone bipolar cell (CL:4033019)
eye 22 : diffuse bipolar 1 cell (CL:4033027)
eye 23 : diffuse bipolar 2 cell (CL:4033028)
eye 24 : diffuse bipolar 3a cell (CL:4033029)
eye 25 : diffuse bipolar 3b cell (CL:4033030)
eye 26 : diffuse bipolar 4 cell (CL:4033031)
eye 27 : diffuse bipolar 6 cell (CL:4033032)
eye 28 : flat midget bipolar cell (CL:4033033)
eye 29 : invaginating midget bipolar cell (CL:4033034)
eye to add : giant bipolar cell (CL:4033035)
eye to add : OFFx cell (CL:4033036)

**Reply by emquardokus** [reply]
Cells and IDs added for 21-29 5-1-2023

**Comment by emquardokus** [comment]
Corrected in the current Eye_v1.3_DRAFT table: https://docs.google.com/spreadsheets/d/1tZ3YZ_PJ5viU2h8Bx3l02uhcMZ41AW4sbmJrKfRtfgg/edit#gid=695483621
**Corrected different labels:**
Several author preferred names were mistakenly the ontology label and vice versa; switched the two.
The other problem was extra spaces at end causing mismatch to ontology.
1. In row 26, the term CL:4033031 removed extra space which was causing mismatch
2.In row 72, the term UBERON:0001783, switch ASCT+B author preferred and label names
3. In row 64, the term UBERON:0004904 switch ASCT+B author preferred and label names
4. In row 74, the term UBERON:0006761 switch ASCT+B author preferred and label names
5. In row 74, the term UBERON:0000482, corrected column for ontology label to basal lamina of epithelium
6. In row 70, the term UBERON:0006136 switch ASCT+B author preferred and label names
7. In row 68, the term UBERON:0034713 switch ASCT+B author preferred and label names
8. In row 34, the term UBERON:0004864 removed space
9. In row 24, the term CL:4033029 removed space

**Comment by emquardokus** [comment]
Corrected in the current Eye_v1.3_DRAFT table: https://docs.google.com/spreadsheets/d/1tZ3YZ_PJ5viU2h8Bx3l02uhcMZ41AW4sbmJrKfRtfgg/edit#gid=695483621

**Blank ontology ID missing parent**
ADDED amacrine cell CL:0000561     In row 16, the term polyaxonal amacrine cell without ontology ID has no parent that is from the CL ontology.
ADDED amacrine cell CL:0000561     In row 16, the term semilunar type 1 cell without ontology ID has no parent that is from the CL ontology.
ADDED amacrine cell CL:0000561     In row 17, the term polyaxonal amacrine cell without ontology ID has no parent that is from the CL ontology.
Problem here is polyaxonal amacrine cell is CT/4 and this is CT/5 and both are new terms: Amacrine cell is listed as CT/3 which covers both: In row 17, the term semilunar type 3 cell without ontology ID has no parent that is from the CL ontology. @anitacaron
ADDED amacrine cell CL:0000561  In row 18, the term polyaxonal amacrine cell without ontology ID has no parent that is from the CL ontology. 

ADDED epithelial fate stem cell	CL:0000036 as parent     In row 74, the term limbal stem cells without ontology ID has no parent that is from the CL ontology.New term request needed for  limbal stem cell defined as Limbal stem cells, also known as corneal epithelial stem cells, are unipotent stem cells located in the basal epithelial layer of the corneal limbus. They form the border between the cornea and the sclera. @aleixpuigb

**Comment by emquardokus** [comment]
Eye_v1.3_DRAFT table: https://docs.google.com/spreadsheets/d/1tZ3YZ_PJ5viU2h8Bx3l02uhcMZ41AW4sbmJrKfRtfgg/edit#gid=695483621

**Different labels**
Found and removed the space!    In row 34, the term UBERON:0004864 has different name/label in the source ontology. The name/label in the ASCT+B table is vasculature of retina _ and the one in the ontology is _vasculature of retina. For reference, the given name/label by SMEs is vasculature. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
**Eye_v1.4_DRAFT**
https://docs.google.com/spreadsheets/d/1tNNPZXL0ycw5gNksDUKHZJpRkKgRdVAUh9AZ6ohpgOA/edit?gid=695483621#gid=695483621

**Different labels**
1.   FIXED The term UBERON:0000965 has a different name/label in the source ontology in the following 2 rows 59, 59. The name/label in the ASCT+B table is lens and the one in the ontology is lens of camera-type eye. For reference, the given name/label by SMEs is lens. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

2. FIXED--rows 64-68 and row 70 this was not mapped correctly--it was to a zebrafish term, changed to optic nerve	cranial nerve II	UBERON:0000941 for vertebrates.    The term UBERON:0004904 has a different name/label in the source ontology in the following 2 rows 70, 70. The name/label in the ASCT+B table is optic nerve and the one in the ontology is neuron projection bundle connecting eye with brain. For reference, the given name/label by SMEs is neuron projection bundle connecting eye with brain. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
**Eye v1.4_DRAFT**
https://docs.google.com/spreadsheets/d/1tNNPZXL0ycw5gNksDUKHZJpRkKgRdVAUh9AZ6ohpgOA/edit?gid=695483621#gid=695483621

**Miscellaneous fixes**
1) Reordered all rows so the AS/2 structures are together to facilitate FTU work.
2) revise Ref column to Ref/n/ID from Ref/n/DOI to accommodate all types of Citations
3) revise format of citations to match implemented support of all citation formats

**Blank ontology ID**
1. Fixed Added  epithelium of conjunctiva	UBERON:0006763 No term id was found for the name/label conjuctival epithelium in the following 1 row 73.
2. Filed NTR after consulting with Aleix:  [NTR]Limbal epithelial stem cell of the cornea #2855 
https://github.com/obophenotype/cell-ontology/issues/2855 No term id was found for the name/label limbal stem cells in the following 1 row 74.
3. Need to discuss inner and outer cortex of lens with experts, after literature review, it seems like this is not clinically recognized.

**Comment by aleixpuigb** [comment]
**New cell types in CL v2025-01-08**

Eye : diffuse bipolar 5 cell (CL:4033085)
Eye : glycinergic-GABAergic amacrine cell (CL:4033091)
Eye : limbal epithelial stem cell of cornea (CL:4033093)
